css/style.css

```diff
@@ -0,0 +1,184 @@
+body{
+	background-color: #ECEEF1;
+	font-family: AvenirNext, Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+.header {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+}
+#account-options {
+	display: block;
+
+}
+.header button img{
+	width: 20px;
+	height: 20px;
+	background-color: #0077C5;
+	padding: 0px 10px 0px 0px;
+}
+.header button {
+	display: flex;
+	background-color: #0077C5;
+	color: white;
+	border-radius: 4px;
+    border: 1px solid #4c4c4c;
+	height: 34px;
+	width: 115px;
+	font-weight: 400;
+	font-size: 13.5;
+	align-items: center;
+	padding: 0px 10px 0px 18px;
+}
+#account-options {
+	font-size: 12px;
+	line-height: 0.5;
+	padding: 0px 7px 0px 0px;
+
+}
+#para {
+	color: #505050;
+}
+a {
+	color: #0077C5;
+	text-decoration: none;
+}
+#intuit{
```

Comment:
Don't use `id` unless required, use `class` instead. This should be applied on all `id` in this CSS file.